### PR TITLE
Add doc

### PR DESCRIPTION
**Pull request by mengniwang95**
## Type of Change

doc

## Description

project name, contact information, example path will be updated later

### CODE REVIEW

File: docs/source/quantization_layer_wise.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+<table class="tg">
+<thead>
+  <tr>
+    <th colspan="2" style="text-align:center;vertical-align:middle">Types/Framework</th>
+    <th style="text-align:center;vertical-align:middle">ONNX Runtime</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td style="text-align:center;vertical-align:middle" colspan="2">W8A8 Post Training Static Quantization</td>
```
Comment (thuang6) [inline docs/source/quantization_layer_wise.md:30]: have we supported LWQ on W8A8 PTSQ in 3.x API?  I remember this is in 2.x feature right?

**Review by thuang6 — COMMENTED** [review]
Assume project name and version will be revised after formal name is finalized

**Review by yiliu30 — COMMENTED** [review]
We need

File: README.md [hunk 2]
```diff
@@ -121,99 +96,36 @@ quantized_model = fit(model=float_model, conf=static_quant_conf, calib_dataloade
...
   <tbody>
     <tr>
-      <td colspan="2" align="center"><a href="./docs/source/design.md#architecture">Architecture</a></td>
-      <td colspan="2" align="center"><a href="./docs/source/design.md#workflow">Workflow</a></td>
-      <td colspan="1" align="center"><a href="https://intel.github.io/neural-compressor/latest/docs/source/api-doc/apis.html">APIs</a></td>
-      <td colspan="1" align="center"><a href="./docs/source/llm_recipes.md">LLMs Recipes</a></td>
-      <td colspan="2" align="center"><a href="examples/README.md">Examples</a></td>
+      <td colspan="3" align="center"><a href="./docs/source/design.md#architecture">Architecture</a></td>
+      <td colspan="3" align="center"><a href="./docs/source/design.md#workflow">Workflow</a></td>
+      <td colspan="3" align="center"><a href="./examples/">Examples</a></td>
```
[the remark this answers is not in the document]
Comment (mengniwang95) [reply README.md:101]: yes, example is in another PR

File: docs/source/quantization_layer_wise.md (hunk 1 above)
Comment (mengniwang95) [reply docs/source/quantization_layer_wise.md:30]: yes, we haven't supported it. let me remove it

File: docs/source/quantization.md [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+    weight_sym=False,
+    weight_group_size=32,
+)
+quant_config.set_local("/h.4/mlp/fc_out/MatMul", fp32_config)
+```
+
+
+### Specify Quantization Backend and Device
+
+Neural-Compressor will detect the hardware and software status automatically to decide which backend should be used. The priority is: accelerator > GPU > CPU.
```
[the remark this answers is not in the document]
Comment (mengniwang95) [reply docs/source/quantization.md:194]: here I mean TRT EP and CUDA EP. if TRT EP is available, CUDA EP will be available at the same time

**Review by yiliu30 — APPROVED** [review]

File: docs/source/SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+Security Policy
+===============
+
+## Report a Vulnerability
+
+Please report security issues or vulnerabilities to the [Intel® Security Center].
```
Comment (chensuyue) [inline docs/source/SECURITY.md:6]: Remove this line.

File: docs/source/imgs/common/intel.svg
Comment (chensuyue) [inline docs/source/imgs/common/intel.svg:1]: Do we need this one? Seems out of date. And better not use Intel Icon in the new Repo.

File: docs/source/installation_guide.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+1. [Installation](#installation)
+
+    1.1. [Prerequisites](#prerequisites)
+
+    1.2. [Install from Binary](#install-from-binary)
+
+    1.3. [Install from Source](#install-from-source)
+
+    1.4. [Install from AI Kit](#install-from-ai-kit)
```
Comment (chensuyue) [inline docs/source/installation_guide.md:11]: Remove this one.

File: docs/source/installation_guide.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+## Installation
+### Prerequisites
+You can install Neural Compressor from binary or source.
+
+The following prerequisites and requirements must be satisfied for a successful installation:
+
+- Python version: 3.8 or 3.9 or 3.10 or 3.11
+
+### Install from Binary
+- Install from Pypi
```
Comment (chensuyue) [inline docs/source/installation_guide.md:28]: Remove this line.

File: docs/source/quantization.md [hunk 7]
```diff
@@ -0,0 +1,245 @@
...
+
+Neural compressor also support to quantize all quantizable ops without accuracy tuning, user can decide whether to tune the model accuracy or not. Please refer to "Get Start" below.
+
+### Working Flow
+
+Currently `accuracy aware tuning` only supports `post training quantization`.
+
+User could refer to below chart to understand the whole tuning flow.
+
+<img src="./imgs/workflow.png" width=600 height=480 alt="accuracy aware tuning working flow">
```
Comment (chensuyue) [inline docs/source/quantization.md:103]: image
Check the scale

File: docs/source/imgs/architecture.png
Comment (thuang6) [inline docs/source/imgs/architecture.png:1]: Better to add AMD CPU, Intel NPU, NV GPU into architecture since it will go into community

**Review by thuang6 — APPROVED** [review]

**Comment by chensuyue** [comment]
The new repo will be https://github.com/onnx/neural-compressor.   
3x ort binary install:   
pip install neural-compressor-ort  
import neural_compressor_ort.quantization

**Pull request merged**